vagrant/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+    # Ubuntu 16.04 LTS
+    config.vm.box = "fosskers/docker-1.12"
+
+    # Docker permissions are available for `ubuntu` user only.
+    config.ssh.username = "ubuntu"
+
+    # Ports to the services
+    config.vm.network :forwarded_port, guest: 8080, host: 8080
+    config.vm.network :forwarded_port, guest: 9999, host: 9999
+    config.vm.network :forwarded_port, guest: 8777, host: 8777
+
+    # VM resource settings
+    config.vm.provider :virtualbox do |vb|
+        vb.memory = 12288
+        vb.cpus = 4
+    end
+
+    # Provisioning
+    config.vm.provision :shell, path: "bootstrap.sh"
```

Comment:
My thoughts are, we should be consistent with approach. The approach with other projects was to use ansible and vagrant, and I see there being a lot of worth in not breaking that pattern without a very clear benefit (which I don't see here).

I want to standardize in line with how our general deployment processes happen, and I'm working on creating some documentation around that. Until then I think it's wise to fall in line with what Hector suggests.